tests/aqdata/aq_base_01.cc

```diff
@@ -1,5 +1,6 @@
 #include "../../src/aqdata/aq_base.h"
 #include "../test_utilities.h"
+
```

Comment:
Delete this empty line. For vertical empty lines, always use one and only one to separate different functional blocks.